api/rds.proto

```diff
@@ -300,7 +300,12 @@ message Route {
     RedirectAction redirect = 3;
   }
 
-  // See base.Metadata description.
+  // See base.Metadata description for message structure. The Metadata field
+  // can be used to provide the Router filter additional information
+  // about the Route. It can be used for configuration, stats, and logging.
+  // The metadata should go under the filter namespace that will need it.
+  // For instance, of the metadata is intended for the Router filter,
+  // the filter name should be specified as "envoy.lb".
```

Comment:
`envoy.lb` is the actual LB, in upstream. It's where things like canary status etc and subset tags get sent. `envoy.router` would be the right namespace for stuff that is actually used by the router filter.